src/prefect/cli/flow_run.py

```diff
@@ -44,21 +45,27 @@ async def inspect(id: UUID):
 
 @flow_run_app.command()
 async def ls(
-    flow_name: List[str] = None,
-    limit: int = 15,
-    state_type: List[StateType] = None,
+    flow_name: List[str] = typer.Option(None, help="Name of the Flow"),
+    limit: int = typer.Option(15, help="Maximum number of Flow Run's to list"),
+    state: List[str] = typer.Option(None, help="Name of the Flow Run's state"),
+    state_type: List[StateType] = typer.Option(
+        None, help="Type of the Flow Run's state"
+    ),
```

Comment:
"Flow" and "Flow Run" should not be capitalized in accordance with our docs styling.